impeller/renderer/backend/gles/capabilities_gles.cc

```diff
@@ -86,6 +86,22 @@ CapabilitiesGLES::CapabilitiesGLES(const ProcTableGLES& gl) {
     gl.GetIntegerv(GL_NUM_SHADER_BINARY_FORMATS, &value);
     num_shader_binary_formats = value;
   }
+
+  {
+    GLint extension_count = 0;
+    gl.GetIntegerv(GL_NUM_EXTENSIONS, &extension_count);
+
+    std::set<std::string> extensions;
```

Comment:
Nit: Why toss it in a set first instead of just `strncmp`ing it iterating over `GetStringi`?

Reply:
Removed, Description does this already